backend/src/test/java/de/amos/apachepulsarui/service/NamespaceServiceIntegrationTest.java

```diff
@@ -61,33 +52,6 @@ void getAllNamespacesOfTenant_returnsNamespacesOfTenant() {
         Assertions.assertThat(namespaceIds).doesNotContain("tenant2/namespace3");
     }
 
-    @Test
-    void getAllNamespaces_returnsMessageCount() throws Exception {
```

Comment:
Würde es Sinn machen den Test woanders unterzubringen? Ans topic vielleicht? : )

Reply:
Ah ja, der sollte auch gar nicht dauerhaft weg 😄 das ist mir durchgerutscht

Reply:
Added the test back to TopicServiceIntegrationTest.